Extension/package.json

```diff
@@ -568,6 +568,12 @@
           "default": true,
           "description": "%c_cpp.configuration.renameRequiresIdentifier.description%",
           "scope": "application"
+        },
+        "C_Cpp.useBacktickCommandSubstitution": {
+          "type": "boolean",
+          "default": false,
```

Comment:
@sean-mcmanus @bobbrow Are you ok with adding this to the c_cpp_properties settings? 

@WardenGnaw FYI

Reply:
It's not a c_cpp_properties.json setting. It's a settings.json. Where else would the setting go?

Reply:
ok. thats fine. Just checking. We don't usually use settings, so i wanted to make sure we aren't stepping on anyone.